### ISSUE TITLE
wand in the barrels/dropper/dispenser. unstashable: true

### ISSUE THREAD

**Issue by Yukyxy** [elBukkit/MagicPlugin#556]
Hello the wand can be put in the barrels 1.14

**Comment by Yukyxy** [comment]
and dropper/ dispenser  too

**Comment by Yukyxy** [comment]
@NathanWolf Hi have you find a solution ?

**Comment by NathanWolf** [comment]
I've just added barrels to the containers list, and fixed the reference to dispensers. Droppers should've been ok as far as I can tell.

This is what the current list looks like, if you find it's wrong you can paste this into plugins/Magic/materials.yml and try making changes.

```
containers:
- chest
- furnace
- brewing_stand
- dispenser
- hopper
- dropper
- trapped_chest
- barrel
```

**Comment by Yukyxy** [comment]
I tried "dropper" "dispensers" it's ok, but no barrels, players can still put their wand
https://zupimages.net/up/19/36/ah7e.png

**Comment by NathanWolf** [comment]
I have fixed barrels in the latest dev build. Thank you for your patience!

**Comment by NathanWolf** [comment]
The fix for this was released in 7.7.5
